views/nodes/show.erb

```diff
@@ -3,25 +3,24 @@
     <a href="/node/<%= @node.id %>/edit">edit</a>
   </h2>
   <p>
-    Project Id: <%= @node.project_id %> <br>
```

Comment:
While i think it's useful for debugging and url forwarding I don't think it's useful for the users.